SampleProblem.rb

```diff
@@ -0,0 +1,51 @@
+exemption = ["chocolate", "cheese", "book", "pills", "tablet", "chocolates"]
+puts "Enter items : "
+items = Array.new
+
+loop do
+  str = gets.chomp
+
+  if str.empty?
+    break
+  end
```

Comment:
Use single line syntax for `if` block if you have one line code.
```suggestion
  break if str.empty?
```